src/services/review/RepoCheckerService.ts

```diff
@@ -578,7 +612,7 @@ export default class RepoCheckerService {
     if (isResource) {
       const resourceRoomName = paths[0]
       const resourceCategoryName = paths[1]
-      const resourceName = paths[2]
+      const resourceName = paths[3]
```

Comment:
this check might be wrong - `isResource` only checks that the permalink doesn't start with an underscore, which doesn't seem strong enough to distinguish between collection/resource pages. 

fwiw, `PageService` has this exact method

Reply:
modified in [6a6fc29](https://github.com/isomerpages/isomercms-backend/pull/1187/commits/6a6fc296ed62a9292c47c50ad3dc0917908c8d85) to use page service instead